index.html

```diff
@@ -258,8 +256,9 @@ <h3 id="verify" data-export="" data-dfn-type="dfn">
   </h3>
   <p>
 Society relies on the integrity of public information.
-We have a responsibility to build web technologies to counter misinformation
-and to maintain the integrity of information for public good.
+We have a responsibility to build web technologies to counter both 
+deliberate and inadvertent attempts to mislead, and to maintain the 
+integrity of information for public good.
```

Comment:
```suggestion
We have a responsibility to build web technologies
to counter attempts to mislead,
deliberate or otherwise,
and to maintain the integrity of information for public good.
```

Take it or leave it, the change is good.